### ISSUE TITLE
`strip_chapter_color` should handle punctuation better

### ISSUE THREAD

**Issue by fortytw2** [rsauvehoover/wandering_inn_scraper#2]
First off, huge thanks for making this! I switched from a scraping tool I had written to this after the site updates. 

Many places currently result in the following:

```text
<Lightskyblue|"|Lightskyblue><Lightskyblue|Look at this ...
```

Around punctuation on colored text. 

I believe this would be pretty straightforward to fix by lifting the color stripping up a little in the codebase, then using soup + adjacency selectors to grab all adjacent runs of a color and collapse them there. Happy to send a PR w/ tests if you'd like.

**Comment by rsauvehoover** [comment]
Yeah, right now the colour tags aren't great. Absolutely feel free to send a PR if you'd like. I'm working on a bit of a bigger refactor (whenever I get to actually working on it) which will change some stuff up but I'm always happy to have people contribute.

Sorry I didn't respond for so long, I missed the GH notification 😬 and haven't been working much on the scraper the last few months
